gammapy/utils/tests/test_metadata.py

```diff
@@ -0,0 +1,37 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from typing import Optional
+import pytest
+from numpy.testing import assert_allclose
+from astropy.coordinates import SkyCoord
+from pydantic import ValidationError
+from gammapy.utils.metadata import CreatorMetaData, MetaData
+
+
+def test_creator():
+    default = CreatorMetaData(date="2022-01-01", creator="gammapy", origin="CTA")
+
+    assert default.creator == "gammapy"
+    assert default.origin == "CTA"
+    assert_allclose(default.date.mjd, 59580)
+
+    default.creator = "other"
+    assert default.creator == "other"
+
+    with pytest.raises(ValidationError):
+        default.date = 3
+
+
+def test_subclass():
```

Comment:
Should we add a test aiming to build a standard class (ie PrintHello) depending on `MetaData` and test the main use case and default created metadata?

Reply:
In the tests an inherited class is tested.